src/Components/PrivateRoute/PrivateRoute.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Route, Redirect } from 'react-router-dom';
+
+
+const PrivateRoute = ({ isLoaded, isAuthenticated, component: Component, ...rest }) => {
+    return (
```

Comment:
Can we create variable for the multiple condition and just to be more explicit?

If not, use one-line-arrow-function